### PR TITLE
Remove unnecesary port

### PR DESCRIPTION
**Pull request by Ark74**
No reverse proxy so it shouldn't have http.

### CODE REVIEW

**Comment by enoch85** [comment]
@Ark74 Great, but there are more places than one that we check if the port is active. Can you please fix? Maybe use this one: `netstat -atunlp | grep 9200`, I don't know which one is better.

**Comment by enoch85** [comment]
The end goal is to check if the service was started and if the port is active as Ubuntu only open ports that are in use.

File: apps/fulltextsearch.sh [hunk 1]
```diff
@@ -68,6 +68,7 @@ check_command echo "deb https://artifacts.elastic.co/packages/$ES_DEB_VERSION.x/
 apt update -q4 & spinner_loading
 apt install elasticsearch=$ES_VERSION -y
 check_command /etc/init.d/elasticsearch start
+apt-mark hold elasticsearch
```
Comment (enoch85) [inline apps/fulltextsearch.sh]: Why?

Comment (enoch85) [reply apps/fulltextsearch.sh]: Aah, just saw the comment. 

But why not release it again?

**Comment by Ark74** [comment]
Please, test  latest commit, seems to be working ok on my side.

**Comment by enoch85** [comment]
Don't have time to test, but we can merge and get feedback.

**Comment by johnczer** [comment]
Hey Daniel,
Has this issue with fulltextsearch been fully resolved?  I am still getting this error even after a fresh install so wasn't sure if there was and underlying issue.
 [Elasticsearch\Common\Exceptions\NoNodesAvailableException]
  No alive nodes found in your cluster


fulltextsearch:index [--output [OUTPUT]]

Sorry but something went wrong. Please report this issue to https://github.com/nextcloud/vm/issues and include the output of the error message. Thank you!
sudo -u www-data php /var/www/nextcloud/occ fulltextsearch:index failed
ncadmin@nextcloud:~$ curl -s 127.0.0.1:9200
ncadmin@nextcloud:~$ curl -XGET localhost:9200
curl: (7) Failed to connect to localhost port 9200: Connection refused

**Comment by enoch85** [comment]
Yes, this was resolved several weeks ago. Please do a fresh install.

**Pull request merged**